src/app/shared/nav/nav-menu.component.ts

```diff
@@ -62,4 +70,12 @@ export class NavMenuComponent {
       }
     });
   }
+
```

Comment:
Really just thinking out loud here. This is excellent work and does exactly what it is intended to. I do find myself wondering if it would be helpful to change at least the begin date to the release date of this episode, maybe even do a switch to hourly with a shorter date range. It's a good idea as a navigation tool to have an easy link to the latest episode performance since it seems that is what users are primarily interested in, just feel like it's falling a little short on the intended goal when most of the chart is often at zero from the default date range. I've often thought the default date range could be based on the release schedule, which might be a bigger idea that we are ready to tackle with the UX.